en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md

```diff
@@ -94,11 +90,6 @@ before `</Handlers>`.
 
 2.  Restart API Manager.
 
-!!! note
-    To perform analytics with the logs, see 
-    [Analyzing the Log Overview]({{base_path}}/observe/api-manager-analytics/analyzing-the-log-overview) .
```

Comment:
Let's point this to monitoring logs section under observe

Reply:
done.